roles/command_line_tools/README.md

```diff
@@ -3,19 +3,21 @@ Xcode Command Line Tools
 
 Install Xcode command line tools.
 
+For macOS >= 12.0, installing the command line tools also installs a Python interpreter.
+
 Requirements
 ------------
 
 None
 
 Role Variables
 --------------
+
 None
 
 Example Playbook
 ----------------
 
-
     - hosts: macs
       roles:
          - samdoran.macos.homebrew
```

Comment:
Looks like maybe it's better to suggest invoking the role with `gather_facts: no` in the example? Also, it seems to be referring to a wrong role.
```suggestion
    - hosts: macs
      gather_facts: no
      roles:
         - samdoran.macos.command_line_tools
```

Bonus points: how about adding a paragraph explaining the implications of enabling `gather_facts`?